docs/DeveloperGuide.md

```diff
@@ -383,6 +383,45 @@ The sequence diagram is as follows:
 
 The exported file can be subsequently imported back into any other instance of ContaX, similar to the existing `.json` system of import/export.
 
+### Enhanced Find Logic
+
+The previous implementation find target is limited to name field only. Hence, extending the find function will increase the user experience by enable them easily search `Person` with `phone`
+`email` and `address` field.
```

Comment:
> increase the user experience

I feel that the phrasing is very off here. Also, the entire paragraph is grammatically incorrect. 
You should not add a break-line in markdown unless absolutely necessary, it does not ignore line-breaks like javadocs.

```suggestion
The previous implementation of the find function is limited to only searching the name field. The enhancement to the feature allows users to search for `Person` objects by more attributes, including by the `phone`, `email` and `address` fields.
```